addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -78,18 +74,23 @@ Depending on your Tellstick model different API methods is available:
 <tr><td>Tellstick Net v.2</td><td></td><td>X</td><td>X</td><td></td></tr>
 <tr><td>Tellstick ZNet Lite v.1</td><td></td><td>X</td><td>X</td><td>X</td></tr>
 <tr><td>Tellstick ZNet Lite v.2</td><td></td><td>X</td><td>X</td><td></td></tr>
-<tr><td><i>Tellstick ZNet Pro (Not released)</i></td><td></td><td>X</td><td>X</td><td></td></tr>
 </table>
 
 #### Telldus Core Bridge
 
+> To enable communication between openhab and tellstick-core service (Telldus center) they must use same architecture, eg. 32-bit or 64-bit. The supplied version from Telldus is compailed against 32-bit architecture. Thereof in combinaiton with better openhab performance using 32-bit Java it's recommended to use 32-bit Java. To check which version currently in use run: `java -d32 -version`
```

Comment:
Consider, "is compiled against..."  "Therefore, it is better to use a 32-bit java in openHAB."  "To check which version of Java is currently in use, run..."